rke/templates/aci.go

```diff
@@ -1,6 +1,52 @@
 package templates
 
 const AciTemplateV500 = `
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Updating the existing template is not supported as will it be used by older versions of ACI that do not support it. You can see an example of adding new template (for Ingress NGINX controller) in this PR https://github.com/rancher/kontainer-driver-metadata/pull/889 and follow the same approach.